src/libraries/sendtohelix.proj

```diff
@@ -78,6 +78,7 @@
 
   <PropertyGroup>
     <TargetsWindows Condition="'$(TargetOS)' == 'Windows_NT'">true</TargetsWindows>
+    <BuildTargetFramework Condition="'$(TestPackages)' != 'true'">$([MSBuild]::ValueOrDefault('$(BuildTargetFramework)', '$(NetCoreAppCurrent)'))</BuildTargetFramework>
```

Comment:
why is this needed?

Reply:
Ah because this project isn't invoked via Build.proj which sets the global property. We might want to add a libraries.helix subset so that we can avoid all these translations (we also hardcode the current tfm in some yml files).

Reply:
I'll open an issue, good idea.

Reply:
Thanks a lot :)